mrp_production_real_cost/models/mrp_production_workcenter_line.py

```diff
@@ -97,7 +97,7 @@ def action_pause(self):
 
     @api.multi
     def action_done(self):
+        result = super(MrpProductionWorkcenterLine, self).action_done()
```

Comment:
But his has been just changed in https://github.com/OCA/manufacture/pull/77 due to the reasons exposed there.